pycds/orm/tables.py

```diff
@@ -607,3 +609,158 @@ class DerivedValue(Base):
             name="obs_derived_value_time_place_variable_unique",
         ),
     )
+
+
+class ClimatologicalPeriod(Base):
+    __tablename__ = "climo_period"
+    id = Column("climo_period_id", Integer, primary_key=True)
+    start_date = Column(DateTime, nullable=False)
+    end_date = Column(DateTime, nullable=False)
+
+
+class ClimatologicalPeriodHistory(Base):
```

Comment:
Inclusion of this and other history tables means that the ORM will not be valid if the schema is only migrated to rev `758b` (support multi climo normals). But that might be a good thing, as we probably never want to rest at that intermediate rev.

Reply:
Is there a way around this issue? Or is it a case of checking out an older tagged version of pycds to work with an older database?

Reply:
The way around this issue is to release an ORM compatible with `758b` (sans hx tracking tables), and another compatible with `7244` (with hx tkg). But I think that will require splitting hx tkg into a separate PR, and release a version after each one.